### PR TITLE
Merge messages

### PR DESCRIPTION
**Pull request by as2388**
Adds pipeline stage for applying codes from the Coda file from show 1 to the answers from the first activation survey.

Will need some modifications to support other shows, but I need to see the coded files for those shows first in order to know how to proceed.

### CODE REVIEW

**Comment by as2388** [comment]
FYI This uses code in Core Data which I still haven't sent a PR for. Will do that now and update this thread when done

**Comment by as2388** [comment]
Required code is in https://github.com/AfricasVoices/CoreDataModules/pull/51

File: messages_merge_coded/messages_merge_coded.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    user = args.user
+    json_input_path = args.json_input_path
+    coded_input_path = args.coded_input_path
+    json_output_path = args.json_output_path
+
+    # Load data from JSON file
+    with open(json_input_path, "r") as f:
+        show_messages = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
+
+    show_number = 1  # TODO: Move to command line arg
```
Comment (lukechurch) [inline messages_merge_coded/messages_merge_coded.py]: GitHub issue?

Comment (as2388) [reply messages_merge_coded/messages_merge_coded.py]: Already in progress on another branch.

**Pull request merged**